yt_dlp/extractor/tiktok.py

```diff
@@ -206,7 +211,11 @@ def extract_addr(addr, add_meta={}):
             'timestamp': int_or_none(aweme_detail.get('create_time')),
             'formats': formats,
             'thumbnails': thumbnails,
-            'duration': int_or_none(traverse_obj(video_info, 'duration', ('download_addr', 'duration')), scale=1000)
+            'duration': int_or_none(traverse_obj(video_info, 'duration', ('download_addr', 'duration')), scale=1000),
+            'availability': self._availability(
+                is_private='Private' in labels,
+                needs_subscription='Friends only' in labels,
+                is_unlisted='Followers only' in labels)
```

Comment:
Is this correct?

Reply:
Context:
![image](https://user-images.githubusercontent.com/20515340/145886913-40110452-72bf-4bd1-a406-1d1106dd6222.png)

Reply:
not really. But unless we want to add new cases, ig this is a good approximation